Junk4Dollars-app/Auctions/AuctionListViewController.swift

```diff
@@ -14,10 +15,11 @@ public class AuctionListViewController: UIViewController {
         super.viewDidLoad()
         self.welcomeLabel?.text = ""
         auctionTableView.dataSource = auctionsDataSource
+        self.presenter = AuctionListPresenter(view: self)
     }
 
     public override func viewWillAppear(_ animated: Bool) {
-        loadAuctions()
+        self.presenter?.loadAuctions()
```

Comment:
Just noticed this, but you have to call `super` whenever you override this method.